packages/node/src/exec/default-attenuator.js

```diff
@@ -45,13 +46,12 @@ export const attenuateModule = (params, originalObject) => {
  *
  * **REMEMBER: The attenuator is _not applied_ to packages without policy!**
  *
- * @param {object} options
- * @param {Partial<LavaMoatPolicy>} [options.policy]
+ * @param {import('../types.js').MakeGlobalsAttenuatorOptions} [options]
```

Comment:
Use 

```js
/**
 * @import {MakeGlobalsAttenuatorOptions} from '../types.js'
 */
```

near the top of the file and just reference `MakeGlobalsAttenuatorOptions` here.

Reply:
fixed db5770c39